bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -648,6 +656,15 @@ type BottlerocketBootstrapContainer struct {
 	UserData string `json:"userData,omitempty"`
 }
 
+// CertBundle holds the cert data for bottlerocket.
```

Comment:
This isn't just for bottlerocket right?
```suggestion
// CertBundle holds the cert data.
```